src/lib/snack-bar/snack-bar.ts

```diff
@@ -140,11 +140,29 @@ export class MdSnackBar {
   /**
    * Creates a new overlay and places it in the correct location.
    */
-  private _createOverlay(): OverlayRef {
+  private _createOverlay(config: MdSnackBarConfig): OverlayRef {
     let state = new OverlayState();
-    state.positionStrategy = this._overlay.position().global()
-        .centerHorizontally()
-        .bottom('0');
+    let positionStrategy = this._overlay.position().global();
+    switch (config.position.horizontal) {
+      case 'left':
```

Comment:
Should be `start` and `end` instead of `left` and `right`, taking the RTL/LTR state into account